security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolationHelper.java

```diff
@@ -26,6 +26,12 @@ private static VoltageLevel getVoltageLevel(LimitViolation limitViolation, Netwo
         Objects.requireNonNull(limitViolation);
 
         Identifiable<?> identifiable = network.getIdentifiable(limitViolation.getSubjectId());
+        if (identifiable == null) {
+            Bus bus = network.getBusView().getBus(limitViolation.getSubjectId());
```

Comment:
External tools such as Hades and DynaFlow will have hard time retrieving the bus name in the NODE_BREAKER case. We need a solution that works both for integrated tools and external tools.

Reply:
well I talk with Didier and I'm coding a solution with an enum I will close this mr and open a new one

Reply:
ok thanks